sdb/command.py

```diff
@@ -493,6 +577,9 @@ class Walker(Command):
     """
     A walker is a command that is designed to iterate over data
     structures that contain arbitrary data types.
+
+    Note: Currently, a Walker cannot be defined to also be a
+          SingleInputCommand.
```

Comment:
I think it might go without saying that multiple inheritance (especially from multiple abstract classes of a single base class) is tricky and being able to do it would be the exception rather than the norm (e.g. inheriting from both PrettyPrinter and Locator, which requires special code in `Locator._call()`, and knowing the order you should inherit from them in).

Put another way, you can't be a Walker, PrettyPrinter, or Locator, and also define your own _call method.  Likewise you can't expect to define your own _call_one method.

I think it's sufficient to explain this in the comment for SingleInputCommand.